RadixWallet/Clients/FactorSourcesClient/FactorSourcesClient+Interface.swift

```diff
@@ -295,8 +295,48 @@ extension FactorSourcesClient {
 	}
 }
 
+extension FactorSourcesClient {
+	enum ShieldFactorStatus {
+		/// A shield can be built from the available Factor Sources
```

Comment:
This should 100 be in Sargon

Reply:
you reviewed before I finished adding my comment 😄 

at first I thought about a generic function that returns the `ShieldFactorStatus` for a given config, but to be honest we will probably only use it for `Config 1.1`, so not worth the trouble probably.

Will move this logic though